admin_manual/configuration_database/linux_database_configuration.rst

```diff
@@ -161,6 +161,23 @@ In case of UTF8MB4 you will also find::
 
     "mysql.utf8mb4" => true,
 
+SSL for MySQL Database
+^^^^^^^^^^^^^^^^^^^^^^
+
+This just covers the SSL database configuration on the Nextcloud server. First you need to configure your database server accordingly.
+
+::
+
+  'dbdriveroptions' => [
+    1007 => '/../ssl-key.pem',
+    1008 => '/../ssl-cert.pem',
+    1009 => '/../ca-cert.pem',
+    1014 => true,
```

Comment:
Let's use the constants directly:

```suggestion
    \PDO::MYSQL_ATTR_SSL_KEY => '/../ssl-key.pem',
    \PDO::MYSQL_ATTR_SSL_CERT => '/../ssl-cert.pem',
    \PDO::MYSQL_ATTR_SSL_CA => '/../ca-cert.pem',
    \PDO::MYSQL_ATTR_SSL_VERIFY_SERVER_CERT => true,
```

Reply:
There was an issue with using the constants directly, it  breaks the usage of ssl 
https://github.com/nextcloud/server/issues/19738

this is why i switched to the "numbers" in the first place.

Reply:
So at least my ssl config, works with the numeric values, with the constants my nc server does not come up...